LoopKit/CarbKit/CarbStore.swift

```diff
@@ -926,7 +926,8 @@ extension CarbStore {
                 completion(.failure(error))
             case .success(let values):
                 guard let value = values.closestPrior(to: date) else {
-                    completion(.failure(.noData))
+                    // If we have no cob values in the store, and did not encounter an error, return 0
+                    completion(.success(CarbValue(startDate: date, quantity: HKQuantity(unit: self.preferredUnit, doubleValue: 0))))
```

Comment:
Maybe `HKQuantity(unit: self.preferredUnit, doubleValue: 0)` -> `HKQuantity(unit: .milligramsPerDeciliter, doubleValue: 0)`; this shouldn't depend on the user's preferred unit, I think. Doesn't really cause any issues, but it's an unneeded dependency that might complicate refactoring?

Reply:
@ps2 this is COB

Reply:
Sorry, I mean .grams, then. :)

Reply:
will update all instances to `.gram()`